src/IssueViz/Editor/IIssueSpanCalculator.cs

```diff
@@ -32,7 +32,7 @@ public interface IIssueSpanCalculator
         /// Returns the text span corresponding to the supplied analysis issue location.
         /// Returns null if the location line hash is different from the snapshot line hash
         /// </summary>
-        SnapshotSpan CalculateSpan(ITextRange range, ITextSnapshot currentSnapshot);
+        SnapshotSpan? CalculateSpan(ITextRange range, ITextSnapshot currentSnapshot);
```

Comment:
The summary comment above needs to be updated (and it looks like it was wrong before because it looks like an EmptySpan was returned if the line hashes didn't match).

It looks like the new behaviour is to return the corresponding span, or `null` if there isn't a `TextRange`, or `EmptySpan` if the `TextRange` can't be found in the document/the line hashes don't match.